metrics/influx.go

```diff
@@ -0,0 +1,90 @@
+package metrics
+
+import (
+	"time"
+    "bytes"
+    "fmt"
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/safe"
+	"github.com/containous/traefik/types"
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics/influx"
+	influxdb "github.com/influxdata/influxdb/client/v2"
+)
+
+var influxClient = influx.New(map[string]string{}, influxdb.BatchPointsConfig{}, kitlog.LoggerFunc(func(keyvals ...interface{}) error {
```

Comment:
why kitlog? traefik use logrus

Reply:
This is dependent on go-kit/metrics/. Even other metrics related code uses the same 
https://github.com/containous/traefik/blob/master/metrics/datadog.go#L9